### PR TITLE
C-9 - Install linters

### PR DESCRIPTION
**Pull request by RubezicM**
initialized npm, added packages, added node_modules folder to .gitignore

### CODE REVIEW

**Comment by ban17** [comment]
Ovo nisam naglasio trebalo bi da pratimo "Code review proces" iz dokumenta https://docs.google.com/document/d/1f8CgYJZWxsWk29sqhR06BG_9p5QzVCmuixgGFlGgn0o/edit#heading=h.f6glervl30iq

Ime pull requesta treba da bude:

C-9 - Install linters
možeš da ga edituješ.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.idea/
\ No newline at end of file
+.idea/
+node_modules/
```
Comment (ban17) [inline .gitignore]: Always add newline at the end

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "codeguides",
+  "version": "1.0.0",
```
Comment (hanii1993) [inline package.json]: Maybe we can use 0.1.0 as first version and use 1.0.0 for first MVP version or something like that

**Review by hanii1993 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jelenabircevic) [reply .gitignore]: Can you add also `.vscode` and `.history/`, for people who use VS Code as their editor?

**Review by ban17 — APPROVED** [review]

**Review by hanii1993 — APPROVED** [review]

**Pull request merged**